generator/src/main/kotlin/io/github/mscheong01/krotodc/util/DescriptorExtensions.kt

```diff
@@ -182,7 +182,7 @@ val Descriptor.toDataClassImport: Import
  */
 val FieldDescriptor.javaFieldName: String
     get() {
-        val jsonName = this.jsonName
+        val jsonName = this.jsonName.snakeToCamelCaseNoRegex()
```

Comment:
In my knowledge, FieldDescriptor.jsonName returns the camel case form and not the original proto field name, so this may be a better fit for your intention
```suggestion
        val jsonName = this.name.snakeToCamelCaseNoRegex()
```

Reply:
You are right, my typo here. Prior to your latest commit, I had created a field named 'protoFieldName' which utilized the 'this.name.snakeToCamelCaseNoRegex()' function.